deeplake/integrations/mmdet/mmdet_.py

```diff
@@ -1216,8 +1219,8 @@ def _train_detector(
 
     if hasattr(model, "CLASSES"):
         warnings.warn("model already has a CLASSES attribute. Will be ignored.")
-
-    model.CLASSES = ds_train[train_labels_tensor].info.class_names
+    if hasattr(ds_train[train_labels_tensor].info, "class_names"):
```

Comment:
shouldn't this be in else statement? If model already has CLASSES, we should be ignoring it based on the warning on the line above

Reply:
fixed it!